### PR TITLE
add patch for use after free in LLVM

### PR DESCRIPTION
**Pull request by vchuravy**
hopefully fixes the issue that has been plaguing building LLVM
in BB 0.2.0, I sofar verified that the patches apply cleanly.

### CODE REVIEW

**Comment by vchuravy** [comment]
Probably should include https://reviews.llvm.org/D72550 as well.

**Comment by vchuravy** [comment]
@staticfloat ready to merge, I build LLVM 7/8/9 locally and verified that things seem to work.

File: 0_RootFS/llvm_common.jl [hunk 1]
```diff
@@ -87,7 +90,9 @@ function llvm_script(;version = v"8.0.1", llvm_build_type = "Release", kwargs...
     CMAKE_FLAGS+=(-DLLVM_ENABLE_CXX1Y=ON -DLLVM_ENABLE_PIC=ON)
 
     # tell libcxx to use compiler-rt
-    CMAKE_FLAGS+=(-DLIBCXX_USE_COMPILER_RT=ON)
+    if [[ "${LLVM_MAJ_VER}" == "9" ]]; then
+        CMAKE_FLAGS+=(-DLIBCXX_USE_COMPILER_RT=ON)
+    fi
```
Comment (staticfloat) [inline 0_RootFS/llvm_common.jl]: Out of curiosity, why do we want to use `compiler-rt` for `libcxx` only on `v9`?

**Comment by staticfloat** [comment]
Will build new LLVMBootstrap shards

**Pull request merged**